libwdi/msapi_utf8.h

```diff
@@ -69,7 +69,7 @@ extern "C" {
 #define isdigitU(c) isdigit((unsigned char)(c))
 #define isspaceU(c) isspace((unsigned char)(c))
 #define isxdigitU(c) isxdigit((unsigned char)(c))
-// NB: other issomething() calls are not implemented as they may require multibyte UTF-8 sequences to be converted
+// NB: other is*() calls are not implemented as they may require multibyte UTF-8 sequences to be converted
```

Comment:
Since this isn't a specific thing, this seems like a reasonable correction.

Fwiw, this is the only correction I made. All other corrections were suggested by Google Sheets's spell checker.